go/file_manager/pfscommon/pfstrace.go

```diff
@@ -0,0 +1,17 @@
+package pfscommon
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func CallerFileLine() string {
+	_, fileName, fileLine, ok := runtime.Caller(2)
```

Comment:
standard library provides file line logging: https://golang.org/pkg/log/#pkg-constants : " Lshortfile                    // final file name element and line number: d.go:23. overrides Llongfile"